Multisig/Logic/Ethereum/GnosisSafe.swift

```diff
@@ -17,27 +17,8 @@ class GnosisSafe {
         case unknown
     }
 
-    var minimumSupportedVersionValue = "1.0.0"
-    var maximumSupportedVersionValue = "1.3.0"
-    private var minimumSupportedVersion: Version { Version(minimumSupportedVersionValue)! }
-    private var maximumSupportedVersion: Version { Version(maximumSupportedVersionValue)! }
```

Comment:
I don't understand why this was deleted. the variables were there for testability

Reply:
True, sorry